mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule KinoExplorer.MixProject do
       {:kino, "~> 0.9.4"},
```

Comment:
Should we do:

```suggestion
      {:kino, "~> 0.9.4 or ~> 0.10.0"},
```

0.10.0 is coming out next week, we might as well get ahead and release a compatible version (and I don't think we broke anything). We might as well do `or ~> 0.10` too. Thoughts?